xml_inscriptions/transcribed/MA.Bos.MFA.G.21.1843.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this file is a duplicate of the (now corrected) over version; I'll keep it for the merge and the replace it myself with the right version.